llvm/test/CodeGen/PowerPC/transparent_union.ll

```diff
@@ -0,0 +1,123 @@
+; RUN: llc -verify-machineinstrs -mtriple=powerpc64le-unknown-linux -mcpu=pwr7 \
```

Comment:
I am not sure the machine-codegen-from-IR test is necessary. We only changed the Clang front-end IR codegen. It stands to reason that this test would have passed even with none of the other changes from this patch.

Reply:
You are right.  I verified that this pass even without this update to clang.  Will remove this IR test from the PR.